libs/video-view/src/index.tsx

```diff
@@ -15,11 +15,15 @@ export type VideoViewProps = {
   displayMuteButton?: boolean;
   displayFullscreenButton?: boolean;
   mediaStream?: MediaStream;
+  url?: string;
```

Comment:
```suggestion
  src?: string;
```

Reply:
Another one is the remote URL. Have you tried this one?  https://streaming.dolby.io/media/lipsynctest.333c28e3.mp4

I think you need to set `corssOrigin` property of video element to support remote file
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/crossOrigin

Also need to observe the network state for errors
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/networkState

Reply:
For the video files, we need to enable loop playback.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/loop

Reply:
I think @vincentsong knows how to fix the CORS issue on the video view - by setting it to `none` or `anonymous`